WordPress/Classes/Utility/Image Dimension Parser/ImageDimensionFetcher.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+
+class ImageDimensionsFetcher: NSObject, URLSessionDataDelegate {
+    // Helpful typealiases for the closures
+    public typealias CompletionHandler = (ImageDimensionFormat, CGSize?) -> Void
+    public typealias ErrorHandler = (Error?) -> Void
+
+    let completionHandler: CompletionHandler
+    let errorHandler: ErrorHandler?
+
+    // Internal use properties
```

Comment:
The comment is redundant (given the `private` keyword)

Reply:
Correct, I like to have a visual separator comment that groups together various properties. I do the same with IBOutlet's and public use properties. I find this helps with scanning the "header" of the class a bit easier.